include/uv.h

```diff
@@ -409,6 +409,37 @@ struct uv_shutdown_s {
   } u;                                                                        \
   UV_HANDLE_PRIVATE_FIELDS                                                    \
 
+#define UV_TIMER_PRIVATE_FIELDS                                               \
```

Comment:
come to think of it, we no longer need these private markers if we directly put the fields in the strctures directly.  though this way may look a bit nicer.  can you check if there are private marked fields that should be extracted?